Lesson 04/lesson04 script3.js

```diff
@@ -0,0 +1,195 @@
+// 1 Вивести на сторінку в один рядок через кому числа від 10 до 20
+
+// let output = ''
+// for (let i = 10; i <= 20; i++) {
+//     output += i + ', ' 
+// }
+// console.log(output)
+
+
+
+
+
+// 2 Вивести квадрати чисел від 10 до 20
+
+// let output = ''
+// for (let i = 10; i <= 20; i++ ) {
+//     output += i**2 + ', ' 
+// }
+// console.log(output)
+
+
+
+
+// 3 Вивести таблицю множення на 7
+
+// let output = ''
+// let result = '\n'
+// for (let i = 1; i <= 10; i++ ) {
+//     output += `7 * ${i} = ${7*i}`
```

Comment:
можна було б просто додати перхід на новий рядок
```suggestion
//     output += `7 * ${i} = ${7*i}\n`
```

Reply:
Дякую, зроблено)